pages/society/[slug]/index.vue

```diff
@@ -11,86 +11,151 @@
+          </template>
+        </UiCarousel>
       </div>
       <div
-        class="m-2 px-2 text-center md:text-left"
-        :class="[productions.length ? 'pt-4' : 'md:w-2/3']"
+        class="lg:col-start-4 lg:col-span-2 lg:row-start-2 lg:row-span-2 lg:sticky lg:top-4"
       >
-        <tip-tap-output :html="society.description" />
-        <div v-if="society.website">
-          <strong>Website: </strong>
-          <a
-            :href="society.website"
-            target="_blank"
-            title="Opens in a new tab"
-            class="text-sta-orange hover:text-sta-orange-dark"
+        <!-- Banner Information Section -->
+        <div class="flex flex-col flex-wrap items-center justify-center">
+          <div
+            v-if="society.logo.url"
+            class="flex justify-center h-40 w-40 pb-4 md:w-60 md:h-60"
           >
-            {{ society.website }}
-          </a>
-        </div>
-        <div v-if="society.contact">
-          <strong>Contact: </strong>
-          <a
-            :href="`mailto:${society.contact}`"
-            target="_blank"
-            title="Opens in a new tab"
-            class="text-sta-orange hover:text-sta-orange-dark"
+            <img
+              ref="society-logo"
+              :src="society.logo.url"
+              :alt="`${society.name} logo`"
+            />
+          </div>
+          <div
+            class="flex justify-center w-full lg:w-3/4 p-4 bg-sta-gray-light rounded-lg"
           >
-            {{ society.contact }}
-          </a>
+            <div>
+              <h2 class="text-sta-orange text-3xl font-semibold">
+                Society Info:
+              </h2>
+              <table class="table-auto mt-2">
+                <tbody>
+                  <tr>
+                    <th class="align-top pb-2 pr-2">Website:</th>
+                    <td v-if="society.website" class="align-top">
+                      <a
+                        :href="society.website"
+                        target="_blank"
+                        title="Opens in a new tab"
+                        class="text-sta-orange hover:text-sta-orange-dark"
+                      >
+                        {{ society.website }}
+                      </a>
+                    </td>
+                    <td v-else class="align-top">No Website Listed</td>
+                  </tr>
+                  <tr>
+                    <th class="align-top pr-2">Contact:</th>
+                    <td v-if="society.contact" class="align-top">
+                      <a
+                        :href="`mailto:${society.contact}`"
+                        target="_blank"
+                        title="Opens in a new tab"
+                        class="text-sta-orange hover:text-sta-orange-dark"
+                      >
+                        {{ society.contact }}
+                      </a>
+                    </td>
+                    <td v-else class="align-top">No Email Listed</td>
```

Comment:
Would it not be nicer just to hide the whole table row if there's no email listed?